### PR TITLE
chore: add new blogs

### PR DESCRIPTION
**Pull request by hacktivist123**
This PR adds new blogs

### CODE REVIEW

File: src/posts/2024-03-27-kubecon-eu-wrap-up-isovalent/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+date: '2024-03-27T17:20:05.000Z'
+title: 'KubeCon Europe 2024 Wrap-Up'
+ogImage: ogimage.webp
+ogSummary: "Isovalent highlight events from the just concluded KubeCon europe with a focus on Cilium's representation at the event"
```
Comment (xmulligan) [inline src/posts/2024-03-27-kubecon-eu-wrap-up-isovalent/index.md:5]:
```suggestion
ogSummary: "Isovalent highlight events from the just concluded KubeCon Europe with a focus on Cilium's representation at the event"
```

File: src/posts/2024-04-03-isovalent-lab-champion/index.md
Comment (xmulligan) [inline src/posts/2024-04-03-isovalent-lab-champion/index.md:1]: this isn't live yet

Comment (xmulligan) [reply src/posts/2024-04-03-isovalent-lab-champion/index.md:1]: My mistake, but look at the comment at the top

Comment (hacktivist123) [reply src/posts/2024-04-03-isovalent-lab-champion/index.md:1]: Resolved it

**Pull request merged**